tests/test_basic_example_v2.py

```diff
@@ -34,13 +34,26 @@
     )
 
 
-@pytest.fixture(scope="session")
-def testmodulev2_venv(pytestconfig) -> Iterator[env.VirtualEnv]:
+@pytest.fixture(scope="function")
+def testmodulev2_editable_install() -> bool:
+    raise NotImplementedError(
+        "Tests using the testmodulev2 should have a marker specifying whether the module "
+        "should be installed in editable mode or not."
+    )
+
+
+@pytest.fixture(scope="function")
+def testmodulev2_venv(
+    testdir: pytest.Testdir,
+    pytestconfig: pytest.Config,
+    testmodulev2_editable_install: bool,
```

Comment:
Wow, does this get its value from the marker of the test that requires the fixture?